.azuredevops/pipelineTemplates/jobs.dependenciesModuleDeployment.yml

```diff
@@ -149,8 +149,8 @@ jobs:
                 Write-Verbose ('Using local tokens [{0}]' -f ($tokenMap.Keys -join ', ')) -Verbose
                 $ConvertTokensInputs.Tokens += $tokenMap
 
-                # Swap 'namePrefix' token if provided as a Azure DevOps variable
-                if(-not [String]::IsNullOrEmpty("$env:TOKEN_NAMEPREFIX")){
+                # Swap 'namePrefix' token if empty and provided as a Azure DevOps variable
+                if([String]::IsNullOrEmpty($ConvertTokensInputs.Tokens['namePrefix'])){
                   Write-Verbose 'Using [namePrefix] token from Azure DevOps Variable Groups' -Verbose
                   $ConvertTokensInputs.Tokens['namePrefix'] = "$env:TOKEN_NAMEPREFIX"
```

Comment:
Wouldn't that need to be `= '$(TOKEN_NAMEPREFIX)'` as per the tenant ID reference above?